monkey/monkey_island/cc/models/test_monkey.py

```diff
@@ -0,0 +1,58 @@
+import uuid
+from datetime import timedelta, datetime
+from time import sleep
+from unittest import TestCase
+
+# noinspection PyUnresolvedReferences
+import mongomock
```

Comment:
Since you're not using this import, I guess there's a reason it needs to be here.
If it's because of the test env case, I think it's better to move this import to the test_env file or even to one of the first files to be loaded

Reply:
Not required for the tests to pass. Deleted. Not sure why I put it there 🤔